core/transaction/entity.go

```diff
@@ -224,7 +224,6 @@ func VerifyTransaction(txnHash string, sharders []string) (*Transaction, error)
 			continue
 		} else {
 			if response.StatusCode != 200 {
-				customError = errors.Wrap(customError, err)
```

Comment:
It is not good idea to skip error. The response error is hidden here. We don't know what happen on api request.

Reply:
It is better to refactor the logic here.

Reply:
Agreed, but that refactoring can go in a different PR. 

I've added this particular line in the previous commit, it wasn't there before.